### PR TITLE
Blog post on beta.2 concurrent editing feature

### LINKED ISSUE #1865: Blog: Nick 24/04/10
### Description

Hey Nick, I was late creating this issue for this month. Marian has a blog going out today, so I am going to switch your spots for this month.

### Epic/Story

_No response_

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

### PR DESCRIPTION
**Pull request by knolleary**
Closes #1865

@gdziuba I'm out until Thursday - I've scheduled this to go out on Wednesday.

You have my blessing for any editorial fixes in my absence 😉 

close https://github.com/FlowFuse/website/issues/1910

### CODE REVIEW

File: src/blog/2024/04/node-red-multiplayer.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: Node-RED Multiplayer mode
+subtitle: An update on our work to bring concurrent editing to Node-RED
+description: An update on our work to bring concurrent editing to Node-RED
+date: 2024-04-10
```
Comment (ZJvandeWeg) [inline src/blog/2024/04/node-red-multiplayer.md:5]:
```suggestion
date: 2024-04-10
image: /blog/2024/04/images/nr-multiplayer.png
```

**Review by ZJvandeWeg — APPROVED** [review]

File: src/blog/2024/04/node-red-multiplayer.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+authors: ["nick-oleary"]
+tags:
+    - posts
+    - flowfuse
+    - node-red
```
Comment (gdziuba) [inline src/blog/2024/04/node-red-multiplayer.md:10]:
```suggestion
    - node-red
    - collaborative editing
    - concurrent editing
```

Comment (gdziuba) [reply src/blog/2024/04/node-red-multiplayer.md:10]: @ZJvandeWeg thoughts on adding this?

**Review by gdziuba — APPROVED** [review]
I am excited for the features.  Great update.

**Comment by gdziuba** [comment]
We need an art request before merging.  Doing that now.

**Comment by ZJvandeWeg** [comment]
As it's been approved by Grey, and has a tile now, I'm going to merge this.

**Pull request merged**